### PR TITLE
add project description

### PR DESCRIPTION
**Pull request by kira-abl**

### CODE REVIEW

**Review by shiryz — COMMENTED** [review]
Looks good, I would suggest adding some stretch goals 
- adding a timer
- showing whether the result is correct
- showing different questions depending on previous answers
- skipping questions

File: week4/Project.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+* As a player, I want to click on a button to start playing and see the first question.
+* As a player, I want to see a multiple choice question with four possible answers.
+* As a player, I want to choose an answer and to see it highlighted (colored, have its box checked, etc.).
+* As a player, I want to click on a button to submit my answer and know whether it's the right one.
+* As a player, I want to know how many questions I have left (e.g., 'Question 2/5'.
+* As a player, I want to receive one point for each correct answer and see my results when the games ends.
+* As a player, I want to be able ti quit the game and return to the home screen.
+
+## Requirements
```
Comment (shiryz) [inline week4/Project.md:16]: Good looking CSS!

**Review by shiryz — CHANGES_REQUESTED** [review]

File: week4/Project.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## These user stories will help you build the app:
+
+* As a player, I want to click on a button to start playing and see the first question.
+* As a player, I want to see a multiple choice question with four possible answers.
+* As a player, I want to choose an answer and to see it highlighted (colored, have its box checked, etc.).
+* As a player, I want to click on a button to submit my answer and know whether it's the right one.
+* As a player, I want to know how many questions I have left (e.g., 'Question 2/5'.
+* As a player, I want to receive one point for each correct answer and see my results when the games ends.
+* As a player, I want to be able ti quit the game and return to the home screen.
```
Comment (jamiecoe) [inline week4/Project.md]:
```suggestion
* As a player, I want to be able to quit the game and return to the home screen.
```

**Comment by kira-abl** [comment]
@shiryz What did you mean by stretch goal number 3? They are supposed to show the player whether the answer is correct (it's a 'normal' goal).

File: week4/Project.md (hunk 2 above)
Comment (jamiecoe) [inline week4/Project.md]: To make it clear that this is a one page app, I'd suggest change this user story slightly:
```suggestion
* As a player, I want to be able to press a button to restart the game
```

**Review by jamiecoe — CHANGES_REQUESTED** [review]

**Review by shiryz — APPROVED** [review]
:sparkles:

**Review by jamiecoe — APPROVED** [review]

**Pull request merged**